samples/cellular/nrf_cloud_multi_service/overlay-coap_nrf_provisioning.conf

```diff
@@ -0,0 +1,40 @@
+# Enable provisioning library/provisioning shell dependencies
+CONFIG_MODEM_ATTEST_TOKEN=y
+
+# Enable the provisioning library
+CONFIG_NRF_PROVISIONING=y
+CONFIG_NRF_PROVISIONING_AUTO_INIT=n
+CONFIG_NRF_PROVISIONING_COAP=y
+
+# Wait for credential installation to be complete before connecting to nRF Cloud
+CONFIG_NRF_CLOUD_CHECK_CREDENTIALS=y
+CONFIG_NRF_CLOUD_CLIENT_ID_SRC_INTERNAL_UUID=y
```

Comment:
do we want this here?
yes, it is needed for auto-onboarding.
but it differs from HTTP provisioning and default MSS builds.

Reply:
Yes, I'm surprised it isn't the selection for HTTP.

Reply:
it should be removed here.
otherwise docs need updated.

Reply:
It will be very soon, but I'll do another PR for that that also updates the REST device message sample.